root/static/scripts/common/utility/isBlank.js

```diff
@@ -1,3 +1,12 @@
-export default function isBlank(str) {
+/*
+ * @flow strict
+ * Copyright (C) 2015 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+export default function isBlank(str: string): boolean {
```

Comment:
I think `str` is intended to allow nullish values here, hence the `|| ''` below.
```suggestion
export default function isBlank(str: ?string): boolean {
```